CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+## 1.12.3 (13.04.2022)
+
+### Enhancements
+
+#### ([1472](https://github.com/allegro/hermes/pull/1472)) Allow matching avro field while defining wiremock stubs
```

Comment:
This is not the only PR merged from version 1.12.2 😄  
![image](https://user-images.githubusercontent.com/76775507/163166587-adc87483-9aa4-4c5e-bdfe-a71dcd82c739.png)

Reply:
🤦‍♂️ You're absolutely right, I've fixed that. Thanks for your insight!